containers/ddev-webserver/Dockerfile

```diff
@@ -104,7 +104,6 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -qq install -o Dpkg::Options::="--for
     less \
     libldap-common \
     libpcre3 \
-    libpq-dev \
```

Comment:
This is for interacting with postgres. I think it's OK to remove, but not sure what it has to do with python leftovers.

Reply:
`libpq-dev` was initially added in the Python PR https://github.com/ddev/ddev/pull/4768/files#diff-131a1297bdf1b5e28b280c2e8e473b47e9f959fe10efd73f421f2e0e058a9364R73

That's why I thought it was necessary for Python only?

Reply:
Works for me.